sonic-net/sonic-buildimage
# [202012] StateDB table cleanup for VNET routes

Added  'VNET_ROUTE_TUNNEL_TABLE', 'VNET_ROUTE_TABLE' to the clean up script.

#### Why I did it
Performing load_minigraph doesn't clear up these tables.
#### How I did it
with my hands...
#### How to verify it
add a tunnel route, do a sudo config load_minigraph and check if the tunnel route is cleared after the reset.
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

Added  'VNET_ROUTE_TUNNEL_TABLE', and 'VNET_ROUTE_TABLE' to the list of tables cleared during sonic reset.
#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
